pivy-importer/src/main/groovy/com/linkedin/python/importer/ivy/IvyFileWriter.groovy

```diff
@@ -98,6 +99,10 @@ class IvyFileWriter {
         return filename.substring(filename.indexOf(version) + version.length() + 1)
     }
 
+    static String getActualModuleName(String filename, String revision) {
```

Comment:
It would be more descriptive to rename so we know where the name comes from. Like  `getModuleNameFromFile` or `getFileName` or something like this

Reply:
Good point, I am adding the source to the name.